crates/sovereign-sdk/module-system/sov-modules-stf-blueprint/src/lib.rs

```diff
@@ -522,6 +522,10 @@ where
             }
         }
 
+        // Sort commitments just in case
+        sequencer_commitments
+            .sort_unstable_by(|c1, c2| c1.l2_end_block_number.cmp(&c2.l2_end_block_number));
```

Comment:
In #983 i am sorting by key
```
sequencer_commitments.sort_by_key(|commitment| commitment.l2_start_block_number);
```

I do think we should use the same sort for consistency